task-list/config/routes.rb

```diff
@@ -2,8 +2,8 @@
   root 'tasks#index'
 
   get 'tasks/new' => 'tasks#new'
+  get 'tasks/:id/edit' => 'tasks#edit', as: :task
```

Comment:
The use of `as: :task` here does make some things easier, I think overall it would be better to avoid using it here specifically. We'll be learning more about these "named routes" later this week.